ivy_models/vit/README.rst

```diff
@@ -50,6 +50,30 @@ Getting started
     # Instantiate vit_h_14 model
     ivy_vit_h_14 = vit_h_14(pretrained=True)
 
+    # Convert the Torch image tensor to an Ivy tensor and adjust dimensions
+    img = ivy.asarray(torch_img.permute((0, 2, 3, 1)), dtype="float32", device="gpu:0")
```

Comment:
Hey! Really sorry for the late review here, it slipped through somehow 😅
Just a small detail, could you please load an image (or the nparray file, whichever is relevant) from the image folder in the repo, and update the returned results if necessary? Thanks!